vtex/faststore
# Feat: Adds `ProductCard` component

## What's the purpose of this pull request?
- [x] Moves `ProductCard` component to @faststore/components 
- [x] Adds `ProductCard` styles on @faststore/ui
- [x] Review `ProductCard` structure

## How to test it?
run `yarn dev` and check the `ProductCard`  component or test it using the doc preview link: [ProductCard Component](https://faststore-site-git-feat-product-card-fs-766-faststore.vercel.app/components/molecules/product-card)

### Starter PR
https://github.com/vtex-sites/nextjs.store/pull/368
